### ISSUE TITLE
Delete elements during Capture Web Page

### ISSUE THREAD

**Issue by DOFfactory** [burtonator/polar-bookshelf#597]
**Is your feature request related to a problem? Please describe.**
There is this lovely addon to Firefox called [PrintFriendly & PDF](https://www.printfriendly.com/) with a genius feature: it does not only capture the text on a webpage but also is able to remove elements from it (mostly ads) by deleting certain `<div>` or `<class>` elements. A user is able to scroll through the text, whole capturing and clicking on the Bin button remove that `<div>` or `<class>` altogether. See screenshot.

**Describe the solution you'd like**
It would be lovely to have this feature in Polar Document Repository (possibly saving space by not keeping unnecessary pictures, as well as enhancing overall readability). Since the whole capture runs on Chrome, maybe worth implementing the plugin?

**Describe alternatives you've considered**
None.

**Additional context**
![screenshot - polar delete element](https://user-images.githubusercontent.com/10846169/51800944-77cd4e00-2237-11e9-9b1c-7a8bb4539f96.png)

**Comment by burtonator** [comment]
We're going to be working on some refactors to Polar so that you can capture directly from the browser.  This would enable a TON of other features without us having to re-implement them in Electron and also support just using that chrome extension directly.

**Comment by DOFfactory** [comment]
Please include all browsers (Edge as well), as at work we are forced to use it. I heard there are some converters that can make a Firefox add-on into an Edge Extension.

**Comment by yoshi314** [comment]
maybe it would be better to just print reduced page to pdf and import into Polar. you've got all the features you need this way.

**Comment by DOFfactory** [comment]
Yeah, sounds like a workaround. The best would be to have the universal browser plugin and import directly, while Polar would make the PDF conversion by itself. Just an idea. BTW, is there a specific location where Polar keeps its files? I was not able to find it.

**Comment by burtonator** [comment]
@DOFfactory 

```Help |  About``` will show you the storage location.

HTML to PDF conversion doesn't really work in practice.  the PDFs are routinely breaking formatting of figures. 

This is why we implemented capture.  We will have either some delete mechanism in Polar natively or support capturing directly from the browser.

We MIGHT be able to have the best of both worlds by sending the raw HTML not just the URL... That way the capturing is mostly already complete and  you just preview it.

**Comment by nitinthewiz** [comment]
I second this request. 

I opened this link - https://theoutline.com/post/7267/living-with-passive-suicidal-ideation in Polar and the CSS is such that the header image becomes really long because of resizing. I tried many different user agents, but the CSS is pretty terrible on this page. I tried to remove the element using Dev Tools, but Polar seems to rewrite the iframe in a strange way.

Steps to reproduce the behavior:
1. Go to https://theoutline.com/post/7267/living-with-passive-suicidal-ideation within Polar
2. Observe how long you need to scroll to get to the content.
